ipwb/replay.py

```diff
@@ -252,9 +254,9 @@ def resolveMemento(urir, datetime):
 def compile_target_uri(url: str, query_string: bytes) -> str:
     """Append GET query string to the page path, to get full URI."""
     if query_string:
-        return '{path}?{query_string}'.format(
-            path=url,
-            query_string=query_string.decode('utf-8'),
+        return '{}?{}'.format(
+            url,
+            query_string.decode('utf-8'),
         )
```

Comment:
With this reduced verbosity it might fit in a single line without complaining about the line length. Does it still have to be broken in multiple lines?

```suggestion
        return '{}?{}'.format(url, query_string.decode('utf-8'))
```